circle-mlir/infra/docker/focal/Dockerfile

```
@@ -197,4 +203,4 @@ RUN echo 'circlemlir ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
 
```

I've confirmed that this modification makes the Python 3.10 `venv` environment take effect in bash.

```suggestion
RUN echo 'source /workdir/venv/bin/activate' >> /etc/bash.bashrc

```

---

```console
shpark@Ubuntu:~/git/ONE/circle-mlir/infra/docker/focal$ docker run --rm -it circle-mlir-focal:latest bash
To run a command as administrator (user "root"), use "sudo <command>".
See "man sudo_root" for details.

circlemlir@8910641875fa:~$ command -v python3
/workdir/venv/bin/python3
circlemlir@8910641875fa:~$ python3 --version
Python 3.10.18
```